Userland/Applications/Piano/PlayerWidget.cpp

```diff
@@ -61,3 +98,17 @@ PlayerWidget::PlayerWidget(TrackManager& manager, AudioPlayerLoop& loop)
 PlayerWidget::~PlayerWidget()
 {
 }
+
+void PlayerWidget::add_track()
+{
+    m_track_manager.add_track();
+    auto latest_track_count = m_track_manager.track_count();
+    auto latest_track_string = String::formatted("{}", latest_track_count);
```

Comment:
Is there not a simpler way of stringifying an int?

Reply:
This did feel like overkill, I found an alternative that I'll use `String::number()`